packages/connect/e2e/__fixtures__/getAddressMultisigPubkeysOrder.ts

```diff
@@ -0,0 +1,121 @@
+import { MultisigPubkeysOrder } from "@trezor/protobuf/src/messages";
+
+// https://github.com/trezor/trezor-firmware/blob/45a029edcc720b0825e4a13504d460faada8b72c/tests/device_tests/bitcoin/test_getaddress.py
+
+const nodeInternal = {
+    // trezorctl btc get-public-node -n m/45h/0
+    node: 'xpub69numS2foCPDWvbnTE95o7m92kWr1CFoSA91EHAADBDHcaQw6VwrDFbg5KUGGnyLdKjJ3ohRG4BHQNzpM1kBbVPKkBQB5j221NcvByaWDz2',
+    address_n: [0, 0],
+};
+
+const nodeExternal = {
+    // trezorctl btc get-public-node -n m/44h/1
+    node: 'xpub6APRH5k5zvDxSyeFvP75fvXS57avzHtHjGiuKoU8PrUU1qASm48F5FURSQ6qqHbqf6126ZSt4bPXqZVfKajbG4QXjtjbDntA38LeofELFtr',
+    address_n: [0, 0],
+};
+
+
+const multisigUnsorted1 = {
+    pubkeys: [
+        nodeInternal,
+        nodeExternal,
+    ],
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.PRESERVED
+};
+
+const multisigUnsorted2 = {
+    pubkeys: [
+        nodeExternal,
+        nodeInternal,
+    ],
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.PRESERVED
+};
+
+const multisigSorted1 = {
+    pubkeys: [
+        nodeInternal,
+        nodeExternal,
+    ],
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.LEXICOGRAPHIC
+};
+
+const multisigSorted2 = {
+    pubkeys: [
+        nodeExternal,
+        nodeInternal,
+    ],
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.LEXICOGRAPHIC
+};
+
+const addressUnsorted1 = '3HnEADzLm88XUugzXmfkfC5Ed6PXK9AXQh';
+const addressUnsorted2 = '3QPLZijs2BepteV7ukcrdB8kDQbA3dXhnx';
+
+
+// trezorctl btc get-address -n m/45h/0/0/7 -m 2 -x xpub1 -x xpub2 --multisig-sort-pubkeys
+export default {
+    method: 'getAddress',
+    setup: {
+        mnemonic: 'mnemonic_all',
+        settings: {
+            safety_checks: 2,
+        },
```

Comment:
It should be run only in new firmwares

Reply:
check on `legacyResults` this is the only way to differentiate by fw version in methods.test.ts since we don't use `conditionalTest` here.

Reply:
Done. :+1: